### DISCUSSION TITLE
【closed】

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#258] [Announcements]
# mp/%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E6%8A%80%E6%9C%AF%E5%9B%A2%E9%98%9F/%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E5%A4%A7%E8%A7%84%E6%A8%A1%E5%A4%9A%E4%BA%91%20CDN%20%E7%AE%A1%E7%90%86%E4%B8%8E%E4%BA%A7%E5%93%81%E5%8C%96%E5%AE%9E%E8%B7%B5-%E6%91%98%E8%A6%81

我订阅的公众号内容存档和摘要

http://localhost:4000/mp/%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E6%8A%80%E6%9C%AF%E5%9B%A2%E9%98%9F/%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E5%A4%A7%E8%A7%84%E6%A8%A1%E5%A4%9A%E4%BA%91%20CDN%20%E7%AE%A1%E7%90%86%E4%B8%8E%E4%BA%A7%E5%93%81%E5%8C%96%E5%AE%9E%E8%B7%B5-%E6%91%98%E8%A6%81.html

**Comment by ATQQ** [comment]
谁直接CV我的 key 在用😒 ，我清理一下 hh

**Reply by yxw007** [reply]
暴露key了

**Reply by ATQQ** [reply]
哈哈 我改一下 给它移动到环境变量里

**Reply by yxw007** [reply]
国内很多这种无脑copy的同学，为了安全最好放进secret中
